package.json

```diff
@@ -141,13 +141,17 @@
       "/tests/",
       "src/common/components",
       "src/common/actions.js",
+      "src/common/serviceUtils.js",
       "src/common/index.js",
       "src/components",
       "src/i18n",
       "src/store",
       "environment.js",
       "src/data",
-      "src/payment/data"
+      "src/payment/data",
+      "src/payment/coupon/data/service",
```

Comment:
Tests for these two files (service and sagas) are forthcoming - we decided to make this PR now and get it in so we can keep moving.